src/assets/scss/components/_table.scss

```diff
@@ -82,7 +82,7 @@ $table-sticky-header-height: 300px !default;
     }
 }
 
-.b-table {
+.o-table {
```

Comment:
Why???

Reply:
I thought it was an typo. Should it stay `b-table`?

Reply:
Yes!!